client/src/pages/Admin/CustomiseResources/index.js

```diff
@@ -0,0 +1,546 @@
+          />
+        </Col>
+
+        <Col w={[4, 6, 6]} mt={4}>
+          <I.BasicInput
+            label="Mental health link 3"
+            type="text"
+            value={MENTAL_HEALTH_3?.value}
+            helper="Enter your preferred mental health link here"
+            handleChange={(value) =>
+              setFormData(value, 'MENTAL_HEALTH_3', 'value')
+            }
+            resetValue={defaultResourcesObj.MENTAL_HEALTH_3.value}
+            error={validationErrs?.MENTAL_HEALTH_3?.value}
+          />
+        </Col>
+        <Col w={[4, 6, 6]} mt={4}>
+          <I.BasicInput
+            label="Mental health button label 3"
+            type="text"
+            helper="Enter your preferred button label here"
+            value={MENTAL_HEALTH_3?.label}
+            handleChange={(value) =>
+              setFormData(value, 'MENTAL_HEALTH_3', 'label')
+            }
+            resetValue={defaultResourcesObj.MENTAL_HEALTH_3.label}
+            error={validationErrs?.MENTAL_HEALTH_3?.label}
+          />
+        </Col>
+
+        <Col w={[4, 6, 6]} mt={4}>
+          <I.BasicInput
+            label="Mental health link 4"
+            type="text"
+            value={MENTAL_HEALTH_4?.value}
+            helper="Enter your preferred mental health link here"
+            handleChange={(value) =>
+              setFormData(value, 'MENTAL_HEALTH_4', 'value')
+            }
+            resetValue={defaultResourcesObj.MENTAL_HEALTH_4.value}
+            error={validationErrs?.MENTAL_HEALTH_4?.value}
+          />
+        </Col>
+        <Col w={[4, 6, 6]} mt={4}>
+          <I.BasicInput
+            label="Mental health button label 4"
+            type="text"
+            helper="Enter your preferred button label here"
+            value={MENTAL_HEALTH_4?.label}
+            handleChange={(value) =>
+              setFormData(value, 'MENTAL_HEALTH_4', 'label')
+            }
+            resetValue={defaultResourcesObj.MENTAL_HEALTH_4.label}
+            error={validationErrs?.MENTAL_HEALTH_4?.label}
+          />
+        </Col>
+      </S.Section>
+      <Row mt={7} style={{ flex: Number(isMobile), alignItems: 'flex-end' }}>
+        <Col w={[4, 6, 4]} style={{ alignItems: 'flex-end' }}>
+          {httpError && (
+            <T.P mb={2} color="error">
+              {httpError}
+            </T.P>
+          )}
+          {validationErrs?.hasError?.length ? (
+            <Col w={[4, 12, 12]}>
+              <T.P mb="2" color="error">
+                At least one of the input fields has not been filled in or
+                details entered incorrectly. Please check the form above for
+                more details.
+              </T.P>
+            </Col>
+          ) : null}
+
+          <Button
+            variant="primary"
+            disabled={false}
+            loading={loading}
+            text="Save changes"
+            type="submit"
+          />
+        </Col>
+      </Row>
```

Comment:
```suggestion
            <Row mt={7} style={{ flex: Number(isMobile), alignItems: 'flex-end' }}>
        {httpError && (
          <Col w={[4, 12, 12]}>
            <T.P mb={2} color="error">
              {httpError}
            </T.P>
          </Col>
        )}
        {validationErrs?.hasError?.length ? (
          <Col w={[4, 12, 12]}>
            <T.P mb="2" color="error">
              At least one of the input fields has not been filled in or details
              entered incorrectly. Please check the form above for more details.
            </T.P>
          </Col>
        ) : null}

        <Col w={[4, 6, 4]}>
          <Button
            variant="primary"
            disabled={false}
            loading={loading}
            text="Save changes"
            type="submit"
          />
        </Col>
      </Row>
```
![image](https://user-images.githubusercontent.com/39485495/202906633-b5964759-b76c-4302-a645-408ae80757f9.png)

Reply:
done